docs/http/traffic-policy/actions/index.mdx

```diff
@@ -9,11 +9,14 @@ pagination_label: Actions
 
 Traffic Policy actions enable you to modify the behavior of traffic flowing through your endpoints. The following actions are available for use on your endpoints:
 
-| Type                               | Description                                            | Supported On      |
-| ---------------------------------- | ------------------------------------------------------ | ----------------- |
-| [custom-response](custom-response) | Send back a pre-defined custom response to the client. | Inbound, Outbound |
-| [deny](deny)                       | Reject incoming traffic to an endpoint.                | Inbound           |
-| [jwt-validation](jwt-validation)   | Validate JSON Web Tokens (JWT).                        | Inbound           |
-| [log](log)                         | Emit an event with custom defined metadata.            | Inbound, Outbound |
-| [rate-limit](rate-limit)           | Rate limit traffic to your upstream servers.           | Inbound           |
-| [url-rewrite](url-rewrite)         | Transform request URLs using regex.                    | Inbound           |
+| Type                               | Description                                                                              | Supported On      |
+| ---------------------------------- | ---------------------------------------------------------------------------------------- | ----------------- |
+| [add-headrs](add-headers)          | Add headers from incoming requests or outgoing responses.                                | Inbound, Outbound |
+| [custom-response](custom-response) | Send back a pre-defined custom response to the client.                                   | Inbound, Outbound |
+| [deny](deny)                       | Reject incoming traffic to an endpoint.                                                  | Inbound           |
+| [jwt-validation](jwt-validation)   | Validate JSON Web Tokens (JWT).                                                          | Inbound           |
+| [log](log)                         | Emit an event with custom defined metadata.                                              | Inbound, Outbound |
+| [rate-limit](rate-limit)           | Rate limit traffic to your upstream servers.                                             | Inbound           |
+| [remove-headrs](remove-headers)    | Remove headers from incoming requests or outgoing responses.                             | Inbound, Outbound |
+| [redirect](redirect)               | Transform your URL using Regular Expressions and redirect users through location header. | Inbound           |
```

Comment:
I don't think regular expressions should be capitalized here, although fine to be overruled.